fetch.bs

```diff
@@ -4076,6 +4159,8 @@ Entries may be removed before that moment arrives.
 <var>request</var> and <var>response</var>, run these steps:
 
 <ol>
+ <li><p>If <var>response</var> is a <a>network error</a>, return failure.
```

Comment:
There's no need for this as a network error is a response without headers so will always fail this check.